project_task_stage_change_restriction/readme/USAGE.md

```
@@ -0,0 +1 @@
+Try to move a task to a restricted stage. If your user doesn't satisfy the stage allowance conditions the following access error will be raised: "Sorry, you are not allowed to move the task '<task_name>' into the stage '<stage_name>'."
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)